suite-native/graph/src/components/Graph.tsx

```diff
@@ -127,7 +127,7 @@ export const Graph = <TGraphPoint extends FiatGraphPoint>({
                 style={applyStyle(graphStyle, { loading, error })}
                 points={points}
                 color={colors.borderSecondary}
-                animated={animated}
+                animated={animated && !error}
```

Comment:
why?

Reply:
I guess no animation is needed when there is error in graph? This doesn't effect the bug that was there, but seemed pointless to do animation below graph error. I can revert it if you think it's better.

Reply:
I woudn't mess with this there might be some edge cases when we will want this.

Reply:
I reverted it here, but might be worth discussing this sometimes to improve the functionality in error state. 
https://github.com/trezor/trezor-suite/pull/11495/commits/364301f34a652fb2940e09fbaf313bb33a3d15c4